tests/baselines/reference/jsdocAbstract2.js

```diff
@@ -0,0 +1,52 @@
+//// [jsdocAbstract2.js]
+/** @abstract */
+class A {
+    /**
+     * @abstract
+     * @returns {number}
+     */
+    foo() {}
+}
+
+class B extends A {
+    foo() {
+        return 1;
+    }
+}
+
+
+
+
+//// [jsdocAbstract2.d.ts]
+/** @abstract */
+declare abstract class A {
+    /**
+     * @abstract
+     * @returns {number}
+     */
+    abstract foo(): number;
+}
+declare class B extends A {
+}
+
+
+//// [DtsFileErrors]
+
+
+tests/cases/conformance/jsdoc/jsdocAbstract2.d.ts(9,15): error TS2515: Non-abstract class 'B' does not implement inherited abstract member 'foo' from class 'A'.
+
+
+==== tests/cases/conformance/jsdoc/jsdocAbstract2.d.ts (1 errors) ====
+    /** @abstract */
+    declare abstract class A {
+        /**
+         * @abstract
+         * @returns {number}
+         */
+        abstract foo(): number;
+    }
+    declare class B extends A {
+                  ~
+!!! error TS2515: Non-abstract class 'B' does not implement inherited abstract member 'foo' from class 'A'.

```

Comment:
This error should not be there. The of `class B` output is missing `foo(): number`